content/webapp/components/IIIFViewer/MainViewer.tsx

```diff
@@ -12,23 +13,60 @@ import { areEqual, FixedSizeList } from 'react-window';
 import styled from 'styled-components';
 
 import { font } from '@weco/common/utils/classnames';
-import { iiifImageTemplate } from '@weco/common/utils/convert-image-uri';
 import LL from '@weco/common/views/components/styled/LL';
 import { useUser } from '@weco/common/views/components/UserProvider/UserProvider';
+import IIIFItem from '@weco/content/components/IIIFItem/IIIFItem';
 import ItemViewerContext, {
   RotatedImage,
 } from '@weco/content/components/ItemViewerContext/ItemViewerContext';
 import useScrollVelocity from '@weco/content/hooks/useScrollVelocity';
-import { fetchCanvasOcr } from '@weco/content/services/iiif/fetch/canvasOcr';
-import { transformCanvasOcr } from '@weco/content/services/iiif/transformers/canvasOcr';
 import { SearchResults } from '@weco/content/services/iiif/types/search/v3';
-import { missingAltTextMessage } from '@weco/content/services/wellcome/catalogue/works';
 import { TransformedCanvas } from '@weco/content/types/manifest';
-import { convertRequestUriToInfoUri } from '@weco/content/utils/iiif/convert-iiif-uri';
 import { TransformedAuthService } from '@weco/content/utils/iiif/v3';
 
 import { queryParamToArrayIndex } from '.';
-import ImageViewer from './ImageViewer';
+
+// Temporary styling for viewer to display audio, video and pdfs
+// will be tidied up in future work
```

Comment:
Is there a ticket we could link to?

Reply:
Not, yet. I'll add a link when I've created it